res/controllers/Pioneer-DDJ-200-scripts.js

```diff
@@ -330,6 +351,12 @@ DDJ200.switchLEDs = function(vDeckNo) {
             0x7F * engine.getValue(vgroup, "pfl"));
     }
 
+    DDJ200.switchHotcueLEDs(vDeckNo);
+};
+
+DDJ200.switchHotcueLEDs = function(vDeckNo) {
```

Comment:
Same here, no need to put it a separate method if it's only used once. That just makes the code harder to follow.

Reply:
I refactored this whole thing to switch only leds that actually changed